     <groupId>me.jvt.hacking</groupId>
     <artifactId>fat-gatling-jar</artifactId>
-    <version>0.4</version>
+    <version>0.5</version>
     <packaging>jar</packaging>

     <properties>
@@ -14,7 +14,7 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <gatling.version>3.0.0</gatling.version>
-        <gatling-plugin.version>3.0.0</gatling-plugin.version>
+        <maven-exec.version>1.2.1</maven-exec.version>
         <scala-maven-plugin.version>3.4.4</scala-maven-plugin.version>
     </properties>

@@ -76,28 +76,36 @@
             </plugin>

             <plugin>
-                <groupId>io.gatling</groupId>
-                <artifactId>gatling-maven-plugin</artifactId>
-                <version>${gatling-plugin.version}</version>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <version>${maven-exec.version}</version>
                 <executions>
                     <execution>
                         <id>BasicSimulation</id>
                         <goals>
-                            <goal>test</goal>
+                            <goal>java</goal>
                         </goals>
-                        <phase>verify</phase>
+                        <phase>integration-test</phase>
                         <configuration>
-                            <simulationClass>me.jvt.hacking.BasicSimulation</simulationClass>
+                            <mainClass>io.gatling.app.Gatling</mainClass>
+                            <arguments>
+                                <argument>-s</argument>
+                                <argument>me.jvt.hacking.BasicSimulation</argument>
+                            </arguments>
                         </configuration>
                     </execution>
                     <execution>
                         <id>OtherSimulation</id>
                         <goals>
-                            <goal>test</goal>
+                            <goal>java</goal>
                         </goals>
-                        <phase>verify</phase>
+                        <phase>integration-test</phase>
                         <configuration>
-                            <simulationClass>me.jvt.hacking.OtherSimulation</simulationClass>
+                            <mainClass>io.gatling.app.Gatling</mainClass>
+                            <arguments>
+                                <argument>-s</argument>
+                                <argument>me.jvt.hacking.OtherSimulation</argument>
+                            </arguments>
                         </configuration>
                     </execution>
                 </executions>
